### PR TITLE
Created passwordquery and hash query functions

### PR DESCRIPTION
**Pull request by bandrel**
Also fixed userquery function. https://github.com/BlackBox-CSP/password-tools/issues/14

### CODE REVIEW

**Comment by ryman1** [comment]
Could you at least give your variables meaningful names if you're not going to document your examples? I wanted to test passwordquery.py, but f1 f2 and f don't mean crap to me, and I don't feel like digging backwards through the code to figure it out. 

"with open(blah) as ntdshashfile" or something similar.

**Comment by bandrel** [comment]
I updated the documentation for you and clarified the variables.  You should be good to test both of the user and password query examples

File: cracked_hash_stats.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def runstats(hcoutput, ntdsdump):
     crackedpws = []
     # Determine the number of unique hashes processed by placing all ntds dump
     # lines in a set.
-    for dumpline in ntdsdump:
```
Comment (ryman1) [inline cracked_hash_stats.py:25]: I assume this was just a find/replace gone wrong or something?

 'f' is not defined anywhere, so this won't run.

Comment (bandrel) [inline cracked_hash_stats.py:25]: Correct

**Comment by ryman1** [comment]
User query isn't working for me.

C:\Python27\python.exe "F:/My Documents/github/password-tools/userquery.py" cracked_ntlm_final.hcoutfile NThashes.out ejarquay.txt
Traceback (most recent call last):
  File "F:/My Documents/github/password-tools/userquery.py", line 14, in <module>
    dic, uncracked = credsfinder.gen_dict(ntdsDumpFile,hashcatfile)
  File "F:\My Documents\github\password-tools\credsfinder.py", line 17, in gen_dict
    username, phash = item.rstrip().split(":")
ValueError: too many values to unpack

Process finished with exit code 1

File: cracked_hash_stats.py (hunk 1 above)
Comment (ryman1) [inline cracked_hash_stats.py:25]: reverted the change f235c07c35564bada386e442d67fdd9d5b4005a9

**Comment by ryman1** [comment]
err that issue seems to be in the master branch as well... looking in to it.

**Comment by ryman1** [comment]
Most of my problems yesterday were user error. I made a couple changes fixes. If you're happy with them, feel free to commit.

**Pull request merged**